build/config.props

```diff
@@ -9,8 +9,8 @@
     <SemanticVersion Condition=" '$(SemanticVersion)' == '' ">$(MajorNuGetVersion).$(MinorNuGetVersion).$(PatchNuGetVersion)</SemanticVersion>
     <VsTargetBranch>lab/d16.$(MinorNuGetVersion)stg</VsTargetBranch>
     <VsTargetBranch Condition="'$(IsEscrowMode)' == 'true'">rel/d16.$(MinorNuGetVersion)</VsTargetBranch>
-    <SdkTargetBranch>release/2.1.4xx</SdkTargetBranch>
-    <CliTargetBranch>release/2.1.4xx</CliTargetBranch>
+    <SdkTargetBranch>release/2.2.1xx</SdkTargetBranch>
```

Comment:
wait for this until my change is merged.

Reply:
2.2.1xx is not correct. 

It's probably gonna be 2.2.2xx at the minimum, as dev ships March, while 2.2.1xx ships November.

Reply:
the Download CLI step of the build script fails with a 404 when trying to set the target branch 2.2.2xx

Reply:
The branch doesn't exist yet. 
I'd imagine we should be targetting master. 

Let me pull up some threads to figure out the correct branch.

Reply:
Target master.

Reply:
Using master targets a 3.0 preview of the CLI/SDK. Most of the tests that fail are just SHA checking, and there are a few "can't find package" errors, but they seem unrelated to the SDK/CLI, so I think it's good to merge.